.github/workflows/docker.yaml

```diff
@@ -46,14 +46,22 @@ jobs:
         uses: docker/build-push-action@v5
         with:
           context: .
-          platforms: linux/amd64,linux/arm64
+          # TODO: enable before merge
+          # platforms: linux/amd64,linux/arm64
           push: true
           tags: ${{ steps.meta.outputs.tags }}
           cache-from: type=gha
           cache-to: type=gha,mode=max
 
   build-openmina-frontend-image:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        configuration:
+          - build_configuration: compose
```

Comment:
Can we have more descriptive configuration name here?

Reply:
Sure, let's do it in another PR, as this is a change on the FE side directly 
cc @directcuteo